app/uk/gov/hmrc/tai/controllers/EmploymentsController.scala

```diff
@@ -36,47 +37,48 @@ import scala.concurrent.ExecutionContext
 class EmploymentsController @Inject() (
   employmentService: EmploymentService,
   authentication: AuthJourney,
-  cc: ControllerComponents
+  cc: ControllerComponents,
+  customErrorHandler: CustomErrorHandler
 )(implicit ec: ExecutionContext)
-    extends BackendController(cc) with ControllerErrorHandler with Logging {
+    extends BackendController(cc) with Logging {
 
   def employments(nino: Nino, year: TaxYear): Action[AnyContent] = authentication.authForEmployeeExpenses.async {
     implicit request =>
       employmentService
         .employmentsAsEitherT(nino, year)
         .bimap(
-          error => errorToResponse(error),
+          error => customErrorHandler.handleControllerErrorStatuses(error),
           employments => Ok(Json.toJson(ApiResponse(EmploymentCollection(employments.employments, None), Nil)))
         )
-        .merge recoverWith taxAccountErrorHandler()
+        .merge recoverWith customErrorHandler.handleControllerExceptions()
```

Comment:
not needed if the error handler is registered as the play error handler in application.conf

Reply:
Fixed